### PR TITLE
% deviation code

### PR DESCRIPTION
**Pull request by sousinha97**
Added percentage deviation as a column, it just tells %increase or %decrease of value between different versions which is not a projection of how much better/worse it performed.

### CODE REVIEW

File: quisby/util.py [hunk 1]
```diff
@@ -85,28 +85,41 @@ def mk_int(string):
     else:
         return 1
 
+def percentage_deviation(item1,item2):
```
Comment (dvalinrh) [inline quisby/util.py:99]: When ever we have a value of 0, we need to flag that as being an issue.   A value of 0 most likely means part of a test did not run properly.

**Review by dvalinrh — CHANGES_REQUESTED** [review]

File: quisby/util.py (hunk 1 above)
Comment (frival) [reply quisby/util.py:99]: I would say "not necessarily".  We do have some benchmarks where a 0% diff is perfectly normal (e.g. pig).  I don't think 0 is necessarily a valid red flag.

File: quisby/benchmarks/linpack/comparison.py [hunk 2]
```diff
@@ -315,13 +315,14 @@ def compare_linpack_results(spreadsheets, spreadsheetId, test_name):
...
                     price_perf.append(float(ele[2]) / float(ele[4]))
-                    price_perf_diff = (
-                        float(price_perf[1]) - float(price_perf[0])
-                    ) / float(price_perf[0])
-
-                    percentage_diff = (float(ele[2]) - float(value[2])) / float(
-                        value[2]
-                    )
+                    price_perf_diff = percentage_deviation(price_perf[0], price_perf[1])
+                    # price_perf_diff = (
```
Comment (frival) [inline quisby/benchmarks/linpack/comparison.py]: I'm assuming you're going to remove the commented out code before final push?

File: quisby/util.py (hunk 1 above)
Comment (frival) [reply quisby/util.py:99]: Err wait, maybe not, I was thinking you were commenting on the % not the individual results.  I agree I can't think of any results that should be 0.0 but at the same time I can't rule it out and I'm not keen on considering something a magic number.

**Review by frival — CHANGES_REQUESTED** [review]
I had a couple of comments, but also one suggestion here: should we rather call it "difference" rather than "deviation" as people tend to think of "standard deviation" when that word is used and this is neither standard deviation nor standard deviation percent.  I'm open to other thoughts though - maybe that's an Americanism?

**Pull request closed without merging**